lambda/src/main/java/com/lambda/IssueUpdater.java

```diff
@@ -57,22 +65,50 @@ public Issue setActualHours(final int issueId) {
                     LocalDateTime.now());
         }
         DecimalFormat df = new DecimalFormat("0.0");
-        float actualHours = Float.parseFloat((df.format(elapsed.toMinutes() / 60)));
+        float actualHours = Float.parseFloat((df.format(elapsed.toMinutes() / 60.0)));
 
         if (actualHours > 999 || actualHours < 0)
             return null;
 
         return client.updateIssue(new UpdateIssueParams(issue.getId()).actualHours(actualHours));
     }
 
+    private Duration calculateWorkingHours(LocalDateTime start, LocalDateTime end) {
+        LocalDateTime current = start;
+        Duration totalWorkingDuration = Duration.ZERO;
+
+        while (current.isBefore(end)) {
+            if (isWeekday(current)) {
+                LocalDateTime endOfDay = current.withHour(19).withMinute(30); // End at 17:30 ITC as 19:30 JST
```

Comment:
Standard time of Viet Nam is ICT
```suggestion
                LocalDateTime endOfDay = current.withHour(19).withMinute(30); // End at 17:30 ICT as 19:30 JST
```